### PR TITLE
EIP-0010: Autolykos v.2 

### PR DESCRIPTION
**Pull request by kushti**
This EIP defines Autolykos v.2 Proof-of-Work scheme

### CODE REVIEW

**Comment by jasondavies** [comment]
I think this is supposed to be EIP-10 - see #15.

**Comment by kushti** [comment]
@jasondavies renamed!

File: eip-0010.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+# Changes
+
+Original Autolykos scheme is decribed at [https://docs.ergoplatform.com/ErgoPow.pdf](https://docs.ergoplatform.com/ErgoPow.pdf).
+Proving and verification functions are defined in the Algorithms 1 and 2 of the paper.
+
+The proposed changes are:
+
+Alg. 2, line 4: m || nonce  => pk || w || m || nonce
+Alg. 2, line 5: H(j|M|pk|m|w) => H(j|pk|w|M|m)
```
Comment (aslesarenko) [inline eip-0010.md:27]: I'm curious, what is the motivation for the change? Why the old version is problematic, and why the new one is the solution?

**Review by greenhat — APPROVED** [review]
